cdap-docs/cdap-apps/source/hydrator/creating.rst

```diff
@@ -46,21 +46,38 @@ configuration for a Batch Application that runs every minute, reading data from
       "config": {
         "schedule": "\* \* \* \* \*",
         "source": {
-          "name": "Stream",
-          "properties": {  
-            "name": "myStream",
-            "duration": "1m"
-          }
+          "name": "streamSource",
+           "plugin": {
```

Comment:
indentation should be consistent. this should line up with the line above, and use 2 spaces instead of 4